pkg/system/rm.go

```diff
@@ -0,0 +1,75 @@
+package system
```

Comment:
Also figured I would mention I added this only after CI was failing (waiting for a `destroy` event that never comes)